assets/js/actions/apollo.js

```diff
@@ -106,12 +60,80 @@ export const setupApolloClient = (getAuthToken, organizationId) => {
       if (Math.ceil(Date.now() / 1000) > store.getState().apollo.tokenClaims.exp) {
         logout()
       }
+
+      if (store.getState().apollo.tokenClaims.exp - Math.ceil(Date.now() / 1000) < 3600) {
+        const newTokenClaims = await getIdTokenClaims()
+        const newAuthLink = setContext((_, { headers }) => {
+          return {
+            headers: {
+              ...headers,
+              authorization:`Bearer ${newTokenClaims.__raw}`,
+              organization: currentOrganizationId
+            }
+          }
+        })
+        const newLink = authErrorLink.concat(authLink.concat(httpLink))
```

Comment:
is this supposed to be `newAuthLink`? `newAuthLink` is never used

Reply:
Ooo yes ty